source/deploy/mobile/host-your-own-push-proxy-service.rst

```diff
@@ -15,6 +15,16 @@ Enable MPNS
 2. Under **Enable Push Notifications**, select **Manually enter Push Notification Service location**.
 3. Enter the location of your MPNS in the **Push Notification Server** field, then select **Save**.
 
+**Deployment recommendations:** 
+
+- Deploy the MPNS inside your private network or in a DMZ, behind your firewall. Ensure the Mattermost server has network access to the MPNS instance.
+- The MPNS does not connect with Mattermost mobile apps directly; the MPNS parses and forwards push notifications from the Mattermost server to the Apple Push Notification Service (APNS) or the Firebase Cloud Messaging (FCM).
+- The MPNS must be able to communicate with the Apple Push Notification Service over HTTP/2. If an outbound proxy appliance is deployed between the MPNS and APNS, ensure it supports HTTP/2.
+- Use encrypted TLS connections between your MPNS and Apple Push Notification Service, between your MPNS and Google FCM, and between your MPNS and your Mattermost server.
+- Ensure that the push proxy can be reached on the correct port. The default port is 8086.
```

Comment:
```suggestion
```

That's the default listening port, but you often deploy it with a reverse proxy in front to use HTTPS. I'd just strike that one as it depends on how they configure the infrastructure.